rmc/resources/grch37/rmc.py

```diff
@@ -0,0 +1,374 @@
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/joint_clinvar_gnomad.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing "population" and "pathogenic" variants.
+
+Table contains common (AF > 0.01) gnomAD variants ("population") and
+ClinVar pathogenic/likely pathogenic missense variants in haploinsufficient genes
+that cause severe disease ("pathogenic") with defined CADD, BLOSUM, Grantham, missense observed/expected ratios,
+missense badness, and PolyPhen-2 scores.
+
+Input to MPC (missense badness, polyphen-2, and constraint) calculations.
+"""
+
+# MPC related resources
+mpc_model_pkl_path = f"{MPC_PREFIX}/{CURRENT_VERSION}/{CURRENT_FREEZE}/mpc_model.pkl"
+"""
+Path to model (stored as pickle) that contains relationship of MPC variables.
+
+Created using logistic regression.
+"""
+
+gnomad_fitted_score = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/gnomad_fitted_scores.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table of gnomAD variants and their fitted scores (from MPC model regression).
+
+Input to MPC (missense badness, polyphen-2, and constraint) calculations on other datasets.
+"""
+
+gnomad_fitted_score_group = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/gnomad_fitted_scores_group.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table of fitted scores for common (AF > 0.01) variants in gnomAD, grouped by score.
+
+Annotated with the total number of variants with and less than each score.
+"""
+
+mpc_release = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/mpc.ht")
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing missense variants in canonical transcripts annotated with MPC.
+"""
+
+mpc_release_dedup = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/mpc_dedup.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing missense variants in canonical transcripts annotated with MPC.
+
+This Table contains only one row per each unique locus/alleles combination.
+"""
+
+# Assessment related resources
+oe_bin_counts_tsv = f"{CONSTRAINT_PREFIX}/{CURRENT_FREEZE}/{CURRENT_FREEZE}/oe_bin.tsv"
+"""
+TSV with RMC regions grouped by obs/exp (OE) bin.
+
+Annotated with proportion coding base pairs, proportion de novo missense (controls),
+proportion de novo missense (case), and proportion ClinVar pathogenic/likely pathogenic
+severe haploinsufficient missense.
+"""
+
+# Reference related resources
```

Comment:
I think this section may belong in another file, it's not strictly RMC/MPC-related

Reply:
true! it was previously used for RMC but we won't use it now. I might just delete this section